packages/headless/src/next/Pagination.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Link from 'next/link';
+
+interface Props {
+  pageInfo: WPGraphQL.WpPageInfo;
+  baseURL: string;
+  nextLabel?: string;
+  nextAriaLabel?: string;
+  previousLabel?: string;
+  previousAriaLabel?: string;
+  ariaLabel?: string;
+}
+
+/**
+ * Pagination component to display a next and previous link on archives.
+ *
+ * @see https://github.com/wpengine/headless-framework/blob/canary/examples/getting-started/wp-templates/category.tsx.
+ */
+export function Pagination({
+  nextLabel = 'Next Page →',
+  nextAriaLabel = 'Next page.',
+  previousLabel = '← Previous Page',
+  previousAriaLabel = 'Previous page.',
+  ariaLabel = 'Pagination',
+  pageInfo,
+  baseURL,
+}: Props): JSX.Element | null {
+  if (!pageInfo.hasNextPage && !pageInfo.hasPreviousPage) {
+    return null;
+  }
+
+  let classes = 'pagination';
+  if (pageInfo.hasNextPage) classes += ' has-next';
+  if (pageInfo.hasPreviousPage) classes += ' has-previous';
+
+  return (
```

Comment:
As it stands right now, the component seems to mostly be a presentational component, with all the business logic living in the example file. This is fine, because people will likely want to use their own presentational component eventually, but I wonder if the business logic then could also be a component so as not to clutter the page template?

Reply:
Added a `categoryOptions` helper in https://github.com/wpengine/headless-framework/pull/154/commits/3b084b9bb8bec883510a6f2d6991562ecb014b46.